### DISCUSSION TITLE
용산 공유오피스 파이브스팟 1개월 이용 후기 - peterkimzz

### DISCUSSION THREAD

**Discussion by giscus** [peterkimzz/blog#20] [Blog Comments]
# 용산 공유오피스 파이브스팟 1개월 이용 후기 - peterkimzz

재직 중인 회사가 여름쯤 장기 재택 근무로 전환했습니다. 저는 입사한지 2년 정도 되었는데 대표님이 위워크를 좋아하셔서 그 동안 위워크 선릉, 종로지점으로 출근했습니다.
재택근무가 길어지니까 아무래도 일 효율이 떨어질 수 밖에 없었고, 회사 일은 둘째 치더라도 집에서 

https://www.peterkimzz.com/fastfive-fivespot-yongsan-review/

**Comment by minemine90** [comment]
peterkimzz 님 안녕하세요, 저는 파이브스팟 매니저 이민해라고 합니다. 우선 이렇게 정성스러운 파이브스팟 이용 후기 남겨주셔서 정말 감사합니다! 파이브스팟은 최근 모바일웹 리뉴얼 작업을 진행하고 있는데요, 해당 페이지 메인에 peterkimzz 님의 후기 링크와 찍어주신 사진을 넣어도 괜찮을지 여쭤보고자 합니다. 괜찮으시다면 작은 감사의 표시로 파이브스팟 멤버십 할인 쿠폰을 제공해드릴 수 있을 것 같습니다. 바쁘시겠지만 검토해보시고 답변 주시면 감사하겠습니다 :) mh.lee@fastfive.co.kr 로 답신주셔도 좋고, 코멘트 남겨주셔도 좋습니다. 그럼 즐거운 하루 보내세요!
